storage/scylladb_identity.h

```diff
@@ -63,45 +63,24 @@ void db_identity_free(db_identity_t** obj);
  * @brief set id in db_identity_t
  *
  * @param[in] obj pointer to db_identity_t
- * @param[in] id id to be set into db_identity_t
+ * @param[in] uuid uuid to be set into db_identity_t
  * @return
  * - SC_OK on success
  * - SC_TA_NULL/SC_STORAGE_INVAILD_INPUT on error
  */
-status_t db_set_identity_id(db_identity_t* obj, cass_int64_t id);
+status_t db_set_identity_uuid(db_identity_t* obj, CassUuid* uuid);
```

Comment:
How about changing `uuid` into `out`?

Reply:
It is used to set value in obj.
I think it is better to declare as `CassUuid`. 
Or Add `const` prefix to declare as a pointer to const `CassUuid` and rename it as `uuid_in` or `pUuid_in`.

Reply:
Never use something like `pUuid_in`. We don't have to specify prefix `p` for pointer types.

Reply:
Using `in` is straightforward.

Reply:
Got it.

Reply:
The reason why [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) existed was because of weak compilers and/or editors. Modern compilers/editors can indicate the data type of each member or parameter. That is, we don't really depend on [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) to figure the types.

Following KISS (Keep It Simple and Stupid) principle, we shall always set shorten-and-still-meaningful argument name.

Reply:
I see. 
Thank a lot!